release notes/v0.53.0.md

```diff
@@ -0,0 +1,173 @@
+
+Given that this can be done with a normal URL, we have decided to drop support for this and have warned users for the last couple of versions.
+
+### Deprecated `k6/experimental/tracing` in favor of a jslib
+
+`k6/experimental/tracing` is arguably not very well named, and there is good chances we would like to use the name for actual trace and span support within k6 in the future.
+
+On top of that it can now be fully supported in js code, which is why [http-instrumentation-tempo
+](https://grafana.com/docs/k6/latest/javascript-api/jslib/http-instrumentation-tempo/) was created.
+
+The jslib is a drop in replacement so all you need to do is replace `k6/experimental/tracing` with `https://jslib.k6.io/http-instrumentation-tempo/1.0.0/index.js`.
+
+The module is planned to be removed in v0.55.0, planned for November 11th, 2024.
+
+### Experimental websockets now require `binaryType` to be set to receive binary messages
+
+As part of the stabilization of the `k6/experimental/websockets` we need to move the default value of `binaryType` to `blob`. It was previously `arraybuffer` and since the last version there was a warning that it needs to be set in order for binary messages to be received.
+
+That warning is now an error.
+
+In the future we will move the default value to `blob` and remove the error.
+
+## New features
+
+The new features include:
+- Native ESM support, which also brings some quality of life JavaScript features
+- Blob support in the experimental websockets module.
+- Experimental OpenTelemetry metrics output.
+- Consolidating cloud related commands and features under `k6 cloud`
+
+### Native ESM support [#3456](https://github.com/grafana/k6/pull/3456)
+
+TODO: expand, talk about additional JavaScript features now support, top-level-await, dynamic import, compatibility-mode history
+
+### Blob support in the experimental websockets module [grafana/xk6-websockets#74](https://github.com/grafana/xk6-websockets/pull/74)
+
+// TODO
+
+### Experimental OpenTelemetry Output [#3834](https://github.com/grafana/k6/pull/3834)
+
+This release introduces a new experimental output for OpenTelemetry. This allows users to send k6 metrics to any OpenTelemetry-compatible backends. More details and usage examples could be found in the [documentation](https://grafana.com/docs/k6/latest/results-output/real-time/opentelemetry/).
+
+To output metrics to OpenTelemetry, use the `experimental-opentelemetry` output option:
+
+```bash
+ k6 run -o experimental-opentelemetry examples/script.js
+```
+
+If you have any feedback or issues, please let us know directly in [the extension repository](https://github.com/grafana/xk6-output-opentelemetry/issues).
+
+### Consolidating cloud features under `k6 cloud` [#3813](https://github.com/grafana/k6/pull/3813)
+
+// TODO
```

Comment:
@oleiade can you please fill this up

Reply:
I think this will need a docs PR as well.